src/test/java/edu/iastate/ece/sd/sdmay2126/input/test_input_json_valid_file.json

```diff
@@ -11,5 +11,6 @@
   "reactionToMaximize": "validReaction",
   "expressionThreshold": 1.0,
   "expressionUncertainty": 1.0,
-  "geneKnockouts": "geneKnockout"
+  "geneKnockouts": "geneKnockout",
+  "customFluxBounds":
```

Comment:
You need a value here or this isn't a valid JSON file. 

`"customFluxBounds": ["one", "two"]`